ynput/ayon-core
# Chore: Move Max client code

## Changelog Description
This PR is to move 3dsMax integration into the client code for the sake of splitting repos for each host in the future.

## Additional info
Test both dev mode and build addon.

## Testing notes:
Dev Mode
1. Build substance painter addon with ```python ./server_addon/create_ayon_addons.py --skip-zip --addon max```
2. Use ```<your_ayon_core_repo>\server_addon\ayon_max\client``` as the addon directory for substance addon and ``<your_ayon_core_repo>\client``` for core addon
3. Launch AYON in dev mode
4. Publishing and Loading should be working as expected.

Build Mode
1. Build substance painter addon with ```python ./server_addon/create_ayon_addons.py --addon max``` and run create_package.py to build core addon 
2. Install substance painter and core addons
3. Launch AYON
4. Publishing and Loading should be working as expected.
